### ISSUE TITLE
Screen flickers when viewing through Facebook's in-app browser

### ISSUE THREAD

**Issue by matikin9** [LACMTA/is-my-bus-changing#11]
May be due to changing CSS via JS, causing an endless loop of screen resizing (https://developer.apple.com/forums/thread/123084).

**Comment by matikin9** [comment]
Possibly due to an Android WebView issue in FB where the content height is too close to the window height.

**Comment by matikin9** [comment]
I think the in-app browser has a bug that triggers specifically when you are at the bottom of a page, you have Messenger installed, and the send via Messenger button + bar is showing (even while at the bottom of the screen).  In that situation, if you attempt to scroll down further, an infinite loop is triggered causing that button + bar to appear and disappear repeatedly, making the whole screen flicker. I thought it was an issue with my site but I used the Chrome dev tools to test and the issue still happens after removing all content from the `<head>` and `<body>`.  I set the body height to be taller than my device's screen and after a few tries to create the right conditions, I still get this flickering issue.
